features/task_validate_image.feature

```diff
@@ -259,3 +260,80 @@ Feature: Verify Enterprise Contract Tekton Tasks
      And the task logs for step "assert" should match the snapshot
      And the task logs for step "report-json" should match the snapshot
      And the task results should match the snapshot
+
+  Scenario: Titles and descriptions can be excluded
+    Given a working namespace
+      And a key pair named "known"
+      And an image named "acceptance/info"
+      And a valid image signature of "acceptance/info" image signed by the "known" key
+      And a valid attestation of "acceptance/info" signed by the "known" key
+      And a cluster policy with content:
+      ```
+      {"publicKey": ${known_PUBLIC_KEY}}
+      ```
+    When version 0.1 of the task named "verify-enterprise-contract" is run with parameters:
+      | IMAGES               | {"components": [{"containerImage": "${REGISTRY}/acceptance/info"}]} |
+      | POLICY_CONFIGURATION | ${NAMESPACE}/${POLICY_NAME}                                         |
+      | IGNORE_REKOR         | true                                                                |
+      | INFO                 | false                                                               |
+    Then the task should succeed
+      And the task logs for step "report" should match the snapshot
+      And the task results should match the snapshot
+
+  Scenario: Effective-time is honored
+    Given a working namespace
+      And a key pair named "known"
+      And an image named "acceptance/effective-time"
+      And a valid image signature of "acceptance/effective-time" image signed by the "known" key
+      And a valid attestation of "acceptance/effective-time" signed by the "known" key
+      And a valid attestation of "acceptance/effective-time" signed by the "known" key
```

Comment:
Are you creating two attestations intentionally, or is this line duplicated?

Reply:
Also line 308 and 327. Maybe add a comment to explain why if it is intentional.

Reply:
Nope, not intentional. My bad. Removed them.